src/main/java/net/canarymod/plugin/Plugin.java

```diff
@@ -151,12 +152,12 @@ public String getPath() {
     }
 
     /**
-     * Gets the {@link Logman} for the Plugin
+     * Gets the {@link Logger} for the Plugin
      *
-     * @return the Plugin's {@link Logman}
+     * @return the Plugin's {@link Logger}
      */
-    public Logman getLogman() {
-        return Logman.getLogman(getName());
+    public Logger getLogger() {
```

Comment:
This is not just likely to break all plugins but will 100% make all old plugins unable to be loaded.

Reply:
Yeah - Canary is going to start following semver, and this will be a major version bump, all plugins should expect to break.

Reply:
Are you on Telegram? Throw me your handle and I'll get you in chat.